hid-lg4ff.c

```diff
@@ -303,10 +312,26 @@ static const struct lg4ff_wheel_ident_info lg4ff_g29_ident_info2 = {
 	USB_DEVICE_ID_LOGITECH_G29_WHEEL
 };
 
+static const struct lg4ff_wheel_ident_info lg4ff_g923_ident_info = {
```

Comment:
The second and third values are the mask and result. Since we're using the same values as the G29 there won't be any match. Appendix A.2 of the Logitech document explains the process to identify the wheel but doesn't cover all models. I'd use the same masks as the G29 but adjust the bcdDevice values to your wheel. This values can be taken from lsusb output. Here use the bcdDevice value when connecting the wheel for the first time without the module switching to native mode.

Reply:
Also, the others modes should be added for the wheel to work correctly in multi-mode.

Reply:
Modes added, bcdDevice changed to real. Unfortunately, wheel does not changing modes with commands...